ov_wag/settings/base.py

```diff
@@ -182,3 +188,8 @@
 
 WAGTAIL_BASE_URL = os.environ.get('OV_BASE_URL')
 WAGTAILADMIN_BASE_URL = os.environ.get('OV_ADMIN_BASE_URL', '')
+
+WAGTAIL_HEADLESS_PREVIEW = {
+    'CLIENT_URLS': {'default': os.environ.get('OV_PREVIEW_URL')},
+    'SERVE_BASE_URL': os.environ.get('OV_BASE_URL', '') + '/preview',
```

Comment:
When running locally, this seems to cause `localhost:8000` to redirect to `localhost:8000/preview`

Reply:
Is `OV_BASE_URL` set to point there? It should point to the frontend.
```
OV_BASE_URL=http://localhost:3000
OV_ADMIN_BASE_URL=http://localhost:8000
OV_PREVIEW_URL=http://localhost:3000/preview
```